sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/jdbc/JdbcConnectionString.java

```diff
@@ -23,43 +25,54 @@ public final class JdbcConnectionString {
         + "supported to enhance authentication with Azure AD by Spring Cloud Azure.";
     public static final String INVALID_PROPERTY_PAIR_FORMAT = "Connection string has invalid key value pair: %s";
     private static final String TOKEN_VALUE_SEPARATOR = "=";
-    private final String jdbcURL;
-    private final Map<String, String> properties = new HashMap<>();
-    private DatabaseType databaseType = null;
+    private final String originalJdbcUrl;
+
+    private final String baseUrl;
+    private final Map<String, String> originalProperties = new TreeMap<>();
```

Comment:
Whether the baseUrl can be replaced by a private method? I think it may simplify the class